src/tests/api/permanent-delete.api.test.ts

```
@@ -0,0 +1,49 @@
+      expect(data.success).toBe(true);
+      expect(data.deletedId).toBe(habitId);
+    });
+
+    it('should handle error if habit does not exist', async () => {
+      global.fetch = vi.fn().mockResolvedValue({
+        ok: false,
+        status: 404,
+        json: async () => ({ error: 'Habit not found' })
+      });
+
+      const response = await fetch(`/api/habits/non-existent-id/permanent-delete`, {
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)